README.md

```diff
@@ -0,0 +1,155 @@
+
+  - Pair - Pair the entire time working linearly together
+  - Pass - 1 person does 1 requirement and then the next person does the next one
+  - Parallel - work on different parts at the same time by agreeing on interfaces and stubs and meeting in the middle
+
+Remember! The goal at The Flatiron School is not to do, it is to *learn*. Make sure you have worked in all three styles of collaboration. We want you to learn how the different styles works, and make sure that together you and your partner understand every part of the code.
+
+## Instructions
+
+### File Structure
+
+```
+├── CONTRIBUTING.md
+├── Gemfile
+├── Gemfile.lock
+├── LICENSE.md
+├── README.md
+├── Rakefile
+├── app
+│   ├── controllers
+│   │   └── application_controller.rb
+│   ├── models
+│   │   ├── tweet.rb
+│   │   └── user.rb
+│   └── views
+│       ├── index.erb
+│       ├── layout.erb
+│       ├── tweets
+│       │   ├── create_tweet.erb
+│       │   ├── edit_tweet.erb
+│       │   ├── show_tweet.erb
+│       │   └── tweets.erb
+│       └── users
+│           ├── create_user.erb
+│           └── login.erb
+├── config
+│   └── environment.rb
+├── config.ru
+├── db
+│   ├── development.sqlite
+│   ├── migrate
+│   │   ├── 20151124191332_create_users.rb
+│   │   └── 20151124191334_create_tweets.rb
+│   ├── schema.rb
+│   └── test.sqlite
+└── spec
+    ├── controllers
+    │   └── application_controller_spec.rb
+    └── spec_helper.rb
+```
+
+### Gemfile and environment.rb
+
+This project is supported by Bundler and includes a `Gemfile`.
+
+Run bundle install before getting started on the project.
+
+As this project has quite a few files, an `environment.rb` is included that loads all the code in your project along with Bundler. You do not ever need to edit this file. When you see require_relative `../config/environment`, that is how your environment and code are loaded.
+
+### Models
+
+You'll need to create two models in `app/models`, one `User` model and one `Tweet`. Both classes should inherit from `ActiveRecord::Base`.
+
+### Migrations
+
+You'll need to create two migrations to create the users and the tweets table.
+
+Users should have a username, email, and password.
+
+Tweets should have content, and a user_id.
+
+### Associations
+
+You'll need to set up the relationship between users and tweets. Think about how the user interacts with the tweets, what belongs to who?
+
+
+### Home Page
+
+You'll need a controller action to load the home page. You'll want to create a view that will eventually link to both a login page and signup page.
+
+### Create Tweet
+
+You'll need to create two controller actions, one to load the create tweet form, and one to process the form submission. The tweet should be created and saved to the database.
```

Comment:
Yeah I think all of this stuff, for the tests we have to tell them the route that we expect things to be at